pages/home.md

```diff
@@ -42,22 +42,18 @@ permalink: /
             </figure>
           </div>
           <div class="column is-offset-1 is-5">
-            <h2 class="title">All of your data, under your control</h2>
+            <h2 class="title">Pods: All your data, under your control</h2>
             <p class="content is-large">
             <a href="{{site.baseurl}}/about">Solid</a> lets people store their 
             data securely in decentralized data stores 
             called <b>Pods</b>. Pods are like secure personal web servers for data.
-            All data in a pod is accessible via the 
-            <a href="https://solid.github.io/specification/" target="_blank">Solid Protocol</a>.
```

Comment:
Not sure why this and the mention of Tim is in a section about Pods.  It just seemed clunky. 
I think if we want to call attention to the specs, that can be its own section.
And a mention of Tim can be part of either the top or bottom blurbs (but in normal size font).

Reply:
My concern is that we continue to hide concrete details about solid, and then end up with the problem we started with: that people read the main page and have no idea what solid actually is.

Reply:
Oh - so, I don't mind if we mention Solid Protocol and link to the specifications - but I think the location is problematic as its appearance in this section is w/o any introduction.
What if at the top after we talk about the web standards, we have a button (to the Solid Protocol/Solid Specifications - do we have a preference?)

Reply:
Added a button "Ongoing Work" to point to https://solid.github.io/specification/ at the top.  See 
https://github.com/solid/solidproject.org/pull/475#discussion_r555289643

Reply:
Actually undid the Ongoing Work button.  Added that as a button to the section on interoperability standards.